src/scss/override/_forms.scss

```diff
@@ -173,96 +173,139 @@ select[multiple].form-control {
 //
 // Checkbox / Radio
 //
-.checkbox label,
-.radio label {
-  padding-left: 30px;
-}
 
-.checkbox input[type=checkbox],
-.radio input[type=radio] {
-  width: 20px;
-  height: 20px;
-  margin-left: -30px;
-
-  &::-ms-check {
-    border-style: solid;
-    display: inline-block;
-    border-width: 2px;
-    background-clip: border-box;
+.form-group {
+  .checkbox:first-of-type,
+  .radio:first-of-type {
+    margin-top: 20px
   }
-}
 
-.checkbox input[type=checkbox] {
-  &::-ms-check {
-    color: $win-color-type-primary;
-    border-color: $win-color-fl-60;
-    background-color: transparent;
+  .form-group-label {
+    font-size: 15px;
+    font-weight: 400;
+    margin-bottom: 0;
+    line-height: 1em;
   }
 
-  &:indeterminate::-ms-check {
-    color: $win-color-type-primary;
-    border-color: $win-color-fl-accent;
-  }
+  margin-bottom: 36px;
 
-  &:checked::-ms-check {
-    color: $win-color-light;
-    background-color: $win-color-fl-accent;
-    border-color: transparent;
-  }
+}
 
-  &:hover::-ms-check {
-    border-color: $win-color-type-primary;
+// Common checkbox and radio styles
+.checkbox,
+.radio {
+  margin: 24px 0;
```

Comment:
are spacing like this defined by Basel?  If so, it would seem there are a fixed set of these various spacings, and that they are responsive, and that they should all be mixins.  the amount of space between two horizontally aligned elements, the amount of margin on an image, etc...